### PR TITLE
[16.0][FIX] dms_field: Avoid access error in dms.access.groups linked to an other company's records

### PR DESCRIPTION
**Pull request by victoralmau**
Avoid access error in `dms.access.groups` linked to an other company's records

Please @pedrobaeza and  @carolinafernandez-tecnativa can you review it?

@Tecnativa TT50390

### CODE REVIEW

**Comment by victoralmau** [comment]
Ping @pedrobaeza and @carolinafernandez-tecnativa

**Review by carolinafernandez-tecnativa — APPROVED** [review]
Functional review LGTM :) Thanks

File: dms_field/models/dms_access_group.py [hunk 1]
```diff
@@ -22,6 +28,15 @@ def _selection_reference_value(self):
         )
         return [(model.model, model.name) for model in models]
 
+    @api.depends("dms_field_ref")
+    def _compute_company_id(self):
+        for item in self.filtered("dms_field_ref"):
```
Comment (pedrobaeza) [inline dms_field/models/dms_access_group.py:34]: For the rest of the records:

```suggestion
        self.company_id = False
        for item in self.filtered("dms_field_ref"):
```

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge minor

**Pull request merged**